disk/datadog_checks/disk/disk.py

```diff
@@ -168,59 +184,59 @@ def _exclude_disk(self, device, file_system, mount_point):
         # account a space might be in the mount point.
         mount_point = mount_point.rsplit(' ', 1)[0]
 
-        return self._partition_blacklisted(device, file_system, mount_point) or not self._partition_whitelisted(
+        return self._partition_excludeed(device, file_system, mount_point) or not self._partition_includeed(
```

Comment:
This should be
```suggestion
        return self._partition_excluded(device, file_system, mount_point) or not self._partition_included(
```
and similarly elsewhere in the file.